.github/workflows/ci.yml

```diff
@@ -15,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        node: [10, 12]
+        node: [12, 14]
```

Comment:
I think limiting default matrix to ubuntu+14(LTS) should be enough. Platform/Node specific tests might be added when really module code needs it.